src/components/Global/styled/components/Form.js

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+import { Form as BS_Form } from "reactstrap";
+
+export default styled(BS_Form)``;
```

Comment:
What's the plan on this component? If this is unfinished should we add like a "TODO: blah blah" comment so we an find these later on?

Reply:
Yeah, that might be a good idea. I might also be able to delete it altogether until someone actually needs it. I left it there from the previous merge.